### ISSUE TITLE
"Not an editor command: silentnew +set\ buftype=nofile" after last update

### ISSUE THREAD

**Issue by Fubukimaru** [jaxbot/github-issues.vim#170]
Hello,

I have seen the last updates and I was willing to try them, however after the upgrade I got the following on :Gissues:

```
Error detected while processing function <SNR>19_showGithubIssues:
line    5:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 219, in showIssueList
  File "<string>", line 240, in printIssueList
vim.error: Vim:E492: Not an editor command: silentnew +set\ buftype=nofile
```

Could it be due to "silentnew" instead of "silent new"?

**Comment by Fubukimaru** [comment]
The problem was located and a potential fix is the pull request #171.

**Comment by osleg** [comment]
Oh my, dunno how did it slipped in

**Comment by osleg** [comment]
Merged in #171
